docgen/src/guide/Migration_guide_v5.md

```diff
@@ -86,6 +89,16 @@ Some of the props has been renamed for a better consistency across the library.
 
 Please refer to [Widgets changes](guide/Migration_guide_v5.html#widgets-changes) & [Connectors changes](guide/Migration_guide_v5.html#connectors-changes) sections for more detail informations.
 
+### Removing deprecation
+
+We introduced a couple of months ago [a warning](https://github.com/algolia/react-instantsearch/commit/59d1cc41f0dc739c658c413a4310d72e64b6832e) about the usage of `searchForFacetValues` in favour of `searchForItems` & `withSearchBox` (now renamed `searchable`). This warning has been removed and so is the legacy API.Update your code if it's not already the case. See below for the list of changes:
+
+* `searchForFacetValues` &rarr; `withSearchBox` &rarr; `searchable` (RefinementList, Menu)
```

Comment:
Why not use directly the real character? Like → (ctrl + cmd + space on mac to trigger the emoji keyboard)